hazelcast/src/main/java/com/hazelcast/nio/IOUtil.java

```diff
@@ -59,9 +63,54 @@
     public static final byte PRIMITIVE_TYPE_DOUBLE = 7;
     public static final byte PRIMITIVE_TYPE_UTF = 8;
 
+    private static final int READ_BUFFER_SIZE = 8192;
+
     private IOUtil() {
     }
 
+
+    public static String fileAsText(File file) throws IOException {
+        FileInputStream stream = null;
+        try {
+            stream = new FileInputStream(file);
+            return toTextFromStream(stream);
+        } finally {
+            closeQuietly(stream);
+        }
+    }
+
+    public static void closeQuietly(Closeable... closeables) {
+        for (Closeable c : closeables) {
+            try {
+                if (c != null) {
+                    c.close();
+                }
+            } catch (IOException e) {
+                EmptyStatement.ignore(e);
+            }
+        }
+    }
+
+     private static String toTextFromStream(InputStream inputStream) throws IOException {
+        InputStreamReader streamReader = null;
+        Reader reader = null;
+        try {
+            streamReader = new InputStreamReader(inputStream, forName("UTF-8"));
+            reader = new BufferedReader(streamReader);
+
+            StringBuilder builder = new StringBuilder();
+            char[] buffer = new char[READ_BUFFER_SIZE];
+            int read;
+            while ((read = reader.read(buffer, 0, buffer.length)) > 0) {
+                builder.append(buffer, 0, read);
+            }
+            return builder.toString();
+        } finally {
+            closeQuietly(reader, streamReader);
```

Comment:
this is unnecessary, isnt it? the `reader.close()` delegates to `streamReader.close()`

Reply:
Also there's already `closeResource(c)` to silently close the IO resources.